win32ss/printing/base/winspool/printers.c

```diff
@@ -834,7 +834,17 @@ IntFixUpDevModeNames( PDOCUMENTPROPERTYHEADER pdphdr )
 
     if (res)
     {
-        FIXME("IFUDMN : Get Printer Name %S\n",pi2->pPrinterName);
+        /* Check if the provided buffer is large enough */
+        DWORD cbDevMode = pi2->pDevMode->dmSize + pi2->pDevMode->dmDriverExtra;
+        if (pdphdr->cbOut < cbDevMode)
+        {
+            res = FALSE;
```

Comment:
I don't think that you should try to copy in this case?

Reply:
Thanks for checking! I indeed missed this during my review

Reply:
If you did a review, I didn't get any comments. Only a "changes requested"